package.json

```diff
@@ -1,38 +1,41 @@
 {
   "name": "jsonforms-react-seed",
-  "version": "0.1.0",
+  "version": "0.1.1",
   "private": true,
   "dependencies": {
-    "@emotion/react": "^11.5.0",
-    "@emotion/styled": "^11.3.0",
     "@jsonforms/core": "3.3.0",
     "@jsonforms/material-renderers": "3.3.0",
     "@jsonforms/react": "3.3.0",
-    "@mui/icons-material": "^5.11.16",
-    "@mui/lab": "^5.0.0-alpha.58",
-    "@mui/material": "^5.13.0",
-    "@mui/styles": "^5.2.3",
-    "@mui/x-date-pickers": "^6.5.0",
-    "@types/node": "^14.14.14",
-    "@types/react": "^17.0.0",
-    "@types/react-dom": "^17.0.0",
-    "react": "^17.0.1",
-    "react-dom": "^17.0.1",
-    "typescript": "~5.2.2"
+    "@mui/icons-material": "^5.15.19",
+    "@mui/lab": "5.0.0-alpha.170",
+    "@mui/material": "^5.15.19",
+    "@mui/x-date-pickers": "^6.x",
+    "@types/node": "^20.14.2",
+    "@types/react": "^18.3.3",
+    "@types/react-dom": "^18.3.0",
```

Comment:
Can we move these to `devDependencies` please?

Reply:
Sure